worker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.14.4-alpine3.12
+
+RUN mkdir "/src"
+WORKDIR "/src"
+
+COPY main.go .
```

Comment:
I don't think copying files like this is going to be easily maintainable; we probably just want to use `COPY . .`

Reply:
Yeah, I actually had that before, can't remember why I changed it now.